### PR TITLE
test: Add bridge on top of bond removal

### PR DESCRIPTION
**Pull request by almusil**

### CODE REVIEW

**Comment by tyll** [comment]
This needs to be properly formatted with black.

**Comment by almusil** [comment]
Done, thanks

File: tests/integration/bond_test.py [hunk 1]
```diff
@@ -366,6 +366,15 @@ def test_create_linux_bridge_over_bond(bond99_with_slave):
         assertlib.assert_state(desired_state)
 
 
+@pytest.mark.xfail(strict=True, reason="Jira issue # NMSTATE-272")
```
Comment (tyll) [inline tests/integration/bond_test.py]: Sorry, there is a small nit I missed before. Please use a full URL to the issue to make it easier to open. Thank you!

Comment (almusil) [reply tests/integration/bond_test.py]: Done

**Pull request merged**